unity/MobiledgeXRestSample/Assets/StatusContainer.cs

```diff
@@ -21,44 +21,44 @@
 using UnityEngine.UI;
 
 /* A (very) simple data container for scrollRect content. */
-public class StatusContainer : MonoBehaviour
-{
-
-  public string status; // Need a list, or accessor.
-  public Text scrollViewText;
-  long count = 0;
-
-  List<string> posts = new List<string>(3);
-
-  // Use this for initialization
-  void Start()
-  {
-    scrollViewText = GameObject.Find("/UICanvas/OutputScrollView/Viewport/Content/GRPCOutputText").GetComponent<Text>();
-  }
-
-  // Update is called once per frame
-  void Update()
-  {
-    status = "[Sample Output]: " + count + "\n";
-    foreach (string post in posts)
-    {
-      status += post + "\n";
-    }
-    count++;
-    scrollViewText.text = status;
-  }
-
-  public void Post(string postText)
-  {
-    if (postText == null)
-    {
-      return;
-    }
-    // Trim first in line if needed.
-    if (posts.Count == posts.Capacity)
-    {
-      posts.RemoveAt(0);
-    }
-    posts.Add(postText);
+public class StatusContainer : MonoBehaviour
+{
+
+  public string status; // Need a list, or accessor.
+  public Text scrollViewText;
+  long count = 0;
+
+  List<string> posts = new List<string>(3);
+
+  // Use this for initialization
+  void Start()
+  {
+    scrollViewText = GameObject.Find("/UICanvas/OutputScrollView/Viewport/Content/GRPCOutputText").GetComponent<Text>();
```

Comment:
Hm, Out of scope, but should be REST now.

Reply:
Noted